evaluator.py

```diff
@@ -54,15 +54,14 @@ def __init__(self, config_path: str):
         self.config = c
         self.test_results = []
 
-    def eval_log(self, log_path: Path):
-
+    def _eval_flow_test(self, run):
         try:
-            with open(log_path, 'r') as file:
-                data = IperfOutput(**json.load(file))
+            data = IperfOutput(**run["flow_test"])
             md = TestMetadata(**data.tft_metadata)
-        except KeyError as e:
-            logger.error(f"KeyError: {e}. Malformed log handed to eval_log()")
-            raise Exception(f"eval_log(): error parsing {log_path} for expected fields")
+        except Exception as e:
```

Comment:
can we catch a more specific exception?

Reply:
Yeah, we could add a catch for both type error and key error which might occur like we did before, but I figured it would be better to handle this more generally since we're just logging and passing the exception along.